src/utils/polymer.dom.html

```diff
@@ -36,6 +36,11 @@
   class EffectiveNodesObserver {
 
     constructor(target, callback) {
+      /** @type {MutationObserver} */
```

Comment:
The `=` in `{MutationObserver=}` is only valid for function parameters. I'm not 100% sure why this is ok in the type system of Closure, but I'd guess that Closure really only tracks assignments. Given that I've only declared the variable, I guess Closure just takes my word for it.